README.md

```diff
@@ -21,17 +21,17 @@ tests:
 
 ### Run tests locally
 
-This program is distributed as a Docker image. To run a container locally:
+This program is distributed as a Docker image. To run a container locally (under a folder contains tests related yml file): 
 ```bash
 docker run --rm \
-    -v $(pwd)/tests.yaml:/tests/tests.yaml \
-    -e "TEST_HOST=example.com" \
+    -v $(pwd)/httpbin.yml:/tests/tests.yml \
+    -e "TEST_HOST=httpbin.org" \
```

Comment:
I would leave this as "example.com", otherwise readers might think there is some connection between the name of the yaml file and the test host.